src/models/skill.ts

```diff
@@ -0,0 +1,13 @@
+import { Category } from "./category"
+
+export class Skill{
+    skillId: number
+    skillName: string
+    category: Category[]
+
```

Comment:
Same as above with skillName and skillId. 
Also, skills should only every be a part of one category so it does not need to be an array